src/components/LeafletMap.js

```diff
@@ -103,6 +102,7 @@ export class LeafletMap {
     dispatch.on("close-infowindow.map", this.handleInfoWindowClose);
     dispatch.on("fetch-map-data-resolve.map", this.handleAddLayer);
     dispatch.on("fetch-map-data-reject.map", this.handleLayerError);
+    dispatch.on("choose-autocomplete-element", this.findAutocompletLocation);
```

Comment:
good use of `dispatch` here. I would recommend namespacing the event (by tacking on `".map"` to the event name here) in case any other components end up needing to respond to it, e.g. `choose-autocomplete-element.map`. See the documentation for [d3-dispatch](https://github.com/d3/d3-dispatch/blob/master/README.md#dispatch_on) on namespacing events for more.